pwndbg/gdblib/nearpc.py

```diff
@@ -162,7 +165,19 @@ def nearpc(pc=None, lines=None, emulate=False, repeat=False) -> List[str]:
             symbol = C.highlight(symbol)
             first_pc = False
 
-        line = " ".join((prefix, address_str, symbol, asm))
+        if show_opcodes_size > 0:
+            opcodes = instr.bytes.hex()
+            opcodes = opcodes[: show_opcodes_size * 2]
+            align = show_opcodes_size * 2 + 10
+            if len(instr.bytes) > show_opcodes_size:
+                opcodes += pwndbg.color.gray("...")
+                align += 9  # len(pwndbg.color.gray(""))
+            opcodes = opcodes.ljust(align, " ")
+            if pwndbg.gdblib.config.highlight_pc and instr.address == pwndbg.gdblib.regs.pc:
+                opcodes = C.highlight(opcodes)
+            line = " ".join((prefix, address_str, opcodes, symbol, asm))
```

Comment:
nit: Instead of duplicating this line here and in the `else` clause, we can probably add a `opcodes = ""` above the if statement and then move this right after the if statement, so both code paths use the same code to construct the output.

Reply:
Sure, I'll remove this `else` clause.